spec/features/merchants/destroy_spec.rb

```diff
@@ -1,11 +1,25 @@
 require 'rails_helper'
 
 RSpec.describe "As a visitor" do
+  before :each do
+    @bike_shop_employee = User.create(name:"Leah", address:"123 Sesame Street", city:"New York", state:"NY", zip:"90210", email: "Leahsocool@gmail.com", password:"Imeanit", password_confirmation:"Imeanit", role: 1)
+
+    visit '/'
+
+    within 'nav' do
+      click_link "Login"
+    end
+
+    fill_in :email, with: @bike_shop_employee.email
+    fill_in  :password, with: @bike_shop_employee.password
+    click_button "Login"
+  end
+  
   describe "When I visit a merchant show page" do
     it "I can delete a merchant" do
       bike_shop = Merchant.create(name: "Brian's Bike Shop", address: '123 Bike Rd.', city: 'Richmond', state: 'VA', zip: 80203)
 
-      visit "merchants/#{bike_shop.id}"
+      visit "/merchants/#{bike_shop.id}"
```

Comment:
Such a little change can make all the difference

Reply:
I'm glad we finally understand why sometimes the forward slash is required and other times not, that namespacing is finally starting to make sense